src/components/FilterBar.vue

```diff
@@ -0,0 +1,197 @@
+                </v-radio-group>
+              </v-card-text>
+              <v-card-actions>
+                <v-btn
+                  color="primary"
+                  :value="submit"
+                  flat
+                  small
+                  @click="applyFilter('status')"
+                >Apply Status</v-btn>
+                <v-btn color="error" :value="submit" flat small @click="clearFilter('status')">Clear</v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-menu>
+        </v-flex>
+
+        <!-- FURNITURE MATERIAL FILTER -->
+        <v-flex pr-3>
+          <v-menu
+            v-model="showMaterial"
+            :close-on-content-click="false"
+            :nudge-width="70"
+            :nudge-bottom="10"
+            offset-y
+          >
+            <template v-slot:activator="{ on }">
+              <v-btn
+                color="primary"
+                round
+                v-on="on"
+                :flat="!matSubmit"
+                :outline="!matSubmit"
+              >Material</v-btn>
+            </template>
+            <v-card>
+              <v-card-text>
+                <v-radio-group v-model="matSelect" :mandatory="false">
+                  <v-radio v-for="t in furnMaterial" :key="t" :label="`${t}`" :value="t"></v-radio>
+                </v-radio-group>
+              </v-card-text>
+              <v-card-actions>
+                <v-btn
+                  color="primary"
+                  :value="submit"
+                  flat
+                  small
+                  @click="applyFilter('material')"
+                >Apply Material</v-btn>
+                <v-btn
+                  color="error"
+                  :value="submit"
+                  flat
+                  small
+                  @click="clearFilter('material')"
+                >Clear</v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-menu>
+        </v-flex>
+      </v-layout>
+    </v-toolbar-items>
+  </v-toolbar>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Prop, Component } from "vue-property-decorator";
+import { FClass, Material, Status, Furniture } from "@/data/Furniture";
+
+@Component
+export default class FilterBar extends Vue {
+  @Prop()
+  search!: string;
+
+  furnType = ["Bed", "Chair", "Couch", "Table", "Dresser"];
+  furnStatus = ["Donor", "On Truck", "Shed", "Delivered", "Unknown"];
+  furnMaterial = ["Wood", "Glass", "Metal", "Plastic"];
+
+  showTypes = false;
+  showStatus = false;
+  showMaterial = false;
+
+  typeSelect = "";
+  statusSelect = "";
+  matSelect = "";
+
+  typeSubmit = false;
+  statusSubmit = false;
+  matSubmit = false;
+
+  // TODO: Should only apply if a value is selected
+  applyFilter(btn: string) {
+    if (btn == "type") {
+      this.typeSubmit = true;
+    } else if (btn == "status") {
+      this.statusSubmit = true;
+    } else if (btn == "material") {
+      this.matSubmit = true;
```

Comment:
Don't forget that you almost always should be checking for equality using `===` in JavaScript/TypeScript.